README.md

```diff
@@ -2,8 +2,11 @@
   <ul>
   <li>DGA4132</li>
   <li>DGA4130</li>
+  <li>TG789vac v3</li>
   <li>TG789vac v2</li>
   <li>TG799vac</li>
+  <li>TG799vac XTREAM</li>
+  <li>TG799vac XTREAM 35b</li>
```

Comment:
the Fastweb's one is called G7**8**9vac XTREAM 35**B**

Reply:
Ok will edit that